README.adoc

```diff
@@ -226,4 +175,79 @@ You must set the `hibernate.hql.bulk_id_strategy` setting in `hibernate.properti
 This is because Hibernate's default behavior (`PersistentTableBulkIdStrategy`) attempts to create intermediate tables to handle delete and update operations on the multiple tables that represent a `JOINED` inheritance hierarchy, but these table creations statements do not conform to Cloud Spanner DDL.
```

Comment:
Can we just make this a default via our `ServiceContributor`?

Reply:
Yup, PR created #151.